lib/cylc/config.py

```diff
@@ -1911,6 +1911,174 @@ def get_graph_raw(self, start_point_string, stop_point_string,
         self._last_graph_raw_edges = graph_raw_edges
         return graph_raw_edges
 
+    def get_graph_edges(self, start_point_string, stop_point_string,
+                        group_nodes=None, ungroup_nodes=None,
+                        ungroup_recursive=False, group_all=False,
+                        ungroup_all=False, is_validate=False):
+        """Convert the abstract graph edges (self.edges, etc) to actual edges
+
+        (This method differs from the get_graph_raw; class attributes are not
+        used to hold information from previous method calls.)
+
+        Actual edges have concrete ranges of cycle points.
+
```

Comment:
Add docstrings for parameters later or a TODO to remind us of doing that later :)